NDTensors/test/blocksparse.jl

```diff
@@ -1,6 +1,7 @@
 using NDTensors
 using LinearAlgebra
 using Test
+using GPUArraysCore
```

Comment:
```suggestion
using GPUArraysCore: @allowscalar
```

Reply:
I don't think that we need these lines in the tests since `using GPUArraysCore: @allowscalar` is now (I just added it) in the `device_lists.jl` file and this file is included in all of the necessary tests. So I removed it from the header of all tests files except `runtests.jl` and `device_list.jl`